snapcraft/storeapi/constants.py

```diff
@@ -25,3 +25,29 @@
 UBUNTU_STORE_API_ROOT_URL = 'https://myapps.developer.ubuntu.com/dev/api/'
 UBUNTU_STORE_SEARCH_ROOT_URL = 'https://search.apps.ubuntu.com/'
 UBUNTU_STORE_UPLOAD_ROOT_URL = 'https://upload.apps.ubuntu.com/'
+UBUNTU_STORE_TOS_URL = 'https://myapps.developer.ubuntu.com/dev/tos/'
+UBUNTU_STORE_ACCOUNT_URL = 'https://myapps.developer.ubuntu.com/dev/account/'
+
+# Messages and warnings.
+MISSING_AGREEMENT = 'Developer has not signed agreement.'
+MISSING_NAMESPACE = 'Developer profile is missing short namespace.'
+AGREEMENT_ERROR = (
+    'You must agree to the developer terms and conditions to upload snaps.')
+NAMESPACE_ERROR = (
+    'You need to set a username. It will appear in the developer field '
+    'alongside the other details for your snap. Please visit {} and login '
+    'again.')
+AGREEMENT_INPUT_MSG = (
+    'Do you agree to the developer terms and conditions. ({})? [y/N]')
```

Comment:
I'm not a fan of mixing formatting in the constants.  It's not a blocker for me just an opinion.